### PR TITLE
Feat: footer component

### PR DESCRIPTION
**Pull request by Nkaka23dev**
###  PR DESCRIPTION

- This PR creates Footer component

### TASKS COMPLETED

- [x] Insert in Footer.jsx
- [x] Update in tailwind config file
- [x] Update pages/index.js

### HOW CAN THIS PR BE TESTED

```
git clone https://github.com/TheGymRwanda/unit8-maroon
git checkout  ft/footer-component
npm install
npm run dev

```

### BEFORE SUBMITTING YOUR PR, ENSURE IT HAS THE FOLLOWING

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard to understand areas
- [x] My changes generate no new warnings

### TICKET NUMBER

#O15: https://www.notion.so/apeunit/Footer-section-bd6cfdd9284c44ff83aac82d9d246e69

### CODE REVIEW

File: components/cards/BlogCard.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Tag from "../ui/Tag";
+
+export default function BlogCard(props) {
+  return (
+    <>
+      <section className="box-border shrink-0 px-5 flex w-75.5 flex-col py-[1.375rem]  rounded-xl border border-solid border-secondaryGrey shadow-2.5xl ipad:w-112 desktop:w-29.8125 desktop:px-35 ">
```
Comment (ishoborabyose) [inline components/cards/BlogCard.jsx:7]: You should move rem values to tailwind config file

File: components/cards/BlogCard.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <p className="w-65.5 font-Neufile text-desktopHeadingTwo font-normal leading-2.5 tracking-cardDescription text-primaryGrey ipad:w-102 desktop:w-27.3125">
+          {props.title}
+        </p>
+       <div className="space-y-[0.75rem] py-1.1 ipad:py-2.1">
```
Comment (ishoborabyose) [inline components/cards/BlogCard.jsx:11]: it is better to define rem values to tailwind config file

**Review by ishoborabyose — CHANGES_REQUESTED** [review]

**Review by Delyc — APPROVED** [review]

File: components/layout/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+                Waldemarstraße 38,
+                10999 Berlin
+            </div>
+            <div className="w-6.625">
+               <Link link="href='https://twitter.com/apeunit'" style="gap-2" text="Twitter" arrow=<Svg /> />
+              <Link link="https://www.facebook.com/apeunit/" style="gap-2" text="FaceBook" arrow=<Svg /> />
+             <Link link="https://www.linkedin.com/company/ape-unit/?originalSubdomain=de"  style="gap-2" text="LinkedIn" arrow=<Svg /> />
+            </div>
+            <p className="w-11.625  ipad:w-13.875 font-Neufile font-normal text-xl ipad:text-2xl text-black tracking-0.0125">Impressum / Privacy</p>
+        </div>
```
Comment (haroldmud) [inline components/layout/Footer.jsx:19]: Replace all the customized breakpoints by the tailwind predefined ones

**Review by haroldmud — APPROVED** [review]
You did a very good job working on this section, however there are still a tiny thing you need to improve for it to look almost perfect...

**Review by ishoborabyose — APPROVED** [review]
# LGTM

**Pull request merged**